cosmoz-moment.html

```diff
@@ -12,10 +31,10 @@
 		window.Cosmoz = window.Cosmoz || {};
 
 		var localSharedMoment = moment(),
```

Comment:
In "Added component and variable description for docs"(bfd23ec8f7b8eaca646ccbb75c920549c66b65a9) that property is removed.